### PR TITLE
feat(index): setup index

### PR DESCRIPTION
**Pull request by davidnuescheler**
general blog indexing setup.

### CODE REVIEW

File: helix-query.yaml [hunk 1]
```diff
@@ -1,2 +1,52 @@
 version: 1
-# See https://main--helix-home--adobe.hlx.page/docs/setup-indexing.html for more information.
+
+indices:
+  blog:
+    source: html
+    fetch: https://{repo}-{owner}.hlx.page/{path}
+    include:
+      - '/blog/*.(docx|md)'
```
Comment (dominique-pfister) [inline helix-query.yaml:8]: So, just files in `/blog` but not underneath, e.g. `/blog/trends/test.docx`?

**Comment by dominique-pfister** [comment]
When I test the index definition on the file `/blog/trends/test.docx`, I get the following index record:
```
{ 
  path: '/blog/test.html',
  author: '',
  title: '6 new photoshop features to make 2021 your most creative ever',
  date: '',
  image: '/blog/media_11aefa96a53b7d056a424098d53a838b65a58a719.png?width=1200&format=pjpg&optimize=medium', 
  description: 'Work smarter, not harder with new Photoshop features designed to anticipate your needs and save you time when working in the app.', 
  sourceHash: 'QE6nyWvVZus8IIQV',
  category: '',
  topics: [],
  lastModified: 1627113285
}
```
Fields `author`, `date`, `category` and `topics` are empty, because their respective `<meta>` elements are missing.

**Review by dominique-pfister — APPROVED** [review]

File: helix-query.yaml (hunk 1 above)
Comment (davidnuescheler) [reply helix-query.yaml]: ah, thanks... yes, that was not intended. better now?

**Comment by davidnuescheler** [comment]
@dominique-pfister thanks for testing. yes the `test.docx` is very outdated. i probably should replace them with some better sample content.

File: helix-query.yaml (hunk 1 above)
Comment (dominique-pfister) [reply helix-query.yaml]: Sorry, no, I had to commit the required change:

`/**/` goes deep

**Pull request merged**